airflow/contrib/hooks/imap_hook.py

```diff
@@ -22,8 +22,9 @@
 import os.path
 import re
 
-from airflow import LoggingMixin, AirflowException
+from airflow import AirflowException
```

Comment:
```suggestion
from airflow.exceptions import AirflowException
```

Reply:
```
from airflow.utils.log.logging_mixin import LoggingMixin
```
LoggingMixin caused problems when it was imported by `airflow`.  In other places there were similar parts of the code, so I corrected them to avoid the problem in the future.